src/prog_models/models/centrifugal_pump.py

```diff
@@ -121,79 +122,92 @@ class CentrifugalPump(prognostics_model.PrognosticsModel):
             'To': 350
         },
 
+        # Wear Rates
+        'wA': 0,
+        'wRadial': 0,
+        'wThrust': 0,
+
         # Initial state
         'x0': {
-            'w': 376.991118431, # 3600 rpm (rad/sec)
+            'w': 376.991118431,  # 3600 rpm (rad/sec)
             'Q': 0,
             'Tt': 290,
             'Tr': 290,
             'To': 290,
             'A': 12.7084,
             'rThrust': 1.4e-6,
             'rRadial': 1.8e-6,
-            'wA': 0.0,
-            'wThrust': 0,
-            'wRadial': 0,
         }
     }
 
+    state_limits = {
```

Comment:
For us, the parameters from the pump that exceed reasonable limits were rThrust and rRadial. Specifically, state estimators occasionally estimated negative values, which led to negative friction coefficients. From our understanding, this leads to more pump rotation meaning less pump friction, which seems incorrect.

Reply:
I agree- we updated the state limits to include both rThrust and rRadial